docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -244,6 +244,46 @@ When I open URL `https://delayed.vividus.dev` in new window
 When I stop page loading
 ----
 
+=== Scroll context
+
+Scrolls the context to an edge
+
+[source,gherkin]
+----
+When I scroll context to $scrollDirection edge
+----
+* `$scrollDirection` - the direction of the scroll. One of:
+** `LEFT` - start of a page/element horizontally
+** `RIGHT` - end of a page/element horizontally
+** `TOP` - start of a page/element vertically
+** `BOTTOM` - end of a page/element vertically
+
+[TIP]
+If the context is not set, the whole page will be scrolled
+
+.Scroll login to a bottom
+[source,gherkin]
+----
+When I change context to element located `id(login)`
+When I scroll context to BOTTOM edge
+----
+
+=== Scroll element into view
+
+Scrolls an element into the view using https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView[JavaScript]
```

Comment:
"JavaScript" doesn't give much context without navigating to the referred page, I mean even a man who knows js will have to navigate instead of just getting this info from link text

Reply:
Any ideas on how to improve?

Reply:
I would remove `using https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView[JavaScript]`, because these are details of the implementation